src/composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "fossbilling/fossbilling",
     "description": "Free and Open Source Billing System",
-    "version": "0.2.10",
```

Comment:
This should remain 0.2.10

Reply:
It was "4.22.0-beta.5" before the PR, updating it without regenerating the lock file throws an error. I don't have a PHP 8.x installation locally so can't really do it.